specutils/fitting/fitmodels.py

```diff
@@ -1,20 +1,214 @@
 from __future__ import division
 
 import operator
+import itertools
 
 import numpy as np
+from scipy.signal import convolve
 import astropy.units as u
-from astropy.modeling import optimizers
+from astropy.stats import sigma_clipped_stats
 
 from ..manipulation.utils import excise_regions
+from ..analysis import fwhm, centroid
 from ..utils import QuantityModel
 from ..manipulation import extract_region
 from ..spectra.spectral_region import SpectralRegion
 from ..spectra.spectrum1d import Spectrum1D
 from astropy.modeling import fitting, Model, models
+from astropy.table import QTable
+
+
+__all__ = ['find_lines_threshold', 'find_lines_derivative', 'fit_lines', 'estimate_parameters']
+
+# Define the initial estimators
+#   This is the default methods to use to estimate astropy model
+#   parameters. This is based on only a small subset of the astropy
+#   models but it was determined that this is a decent start as most
+#   fitting will probably use one of these.
+#
+#   Each method list must take a Spectrum1D object and should return
+#   a Quantity.
+
+parameter_estimators = {
+    'Gaussian1D': {
+        'amplitude': lambda s: max(s.flux),
+        'mean': lambda s: centroid(s, region=None),
+        'stddev': lambda s: fwhm(s) 
+    },
+    'Lorentzian1D': {
+        'amplitude': None,
```

Comment:
Am I right that you're thinking to do a follow-on to add these?

Reply:
Oh, weird, I had added those. Huh, don't know what happened.  Will add (again)

Reply:
added estimators for each of them.